Day2ConsoleAppHomework/Program.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Day2ConsoleAppHomework
 {
+    enum TransactionType { Buy, Sell }
+
     class Program
     {
+        string currency = "DKK";
```

Comment:
if the var is not changed pls make it a constant ( const string currency = " DKK" )

Reply:
Ok.